rear_left_camera_node:
  ros__parameters:
    camera_frame: rear_left_camera_link
    camera_optical_frame: rear_left_camera_optical_link
    vertical_fov: 2.0978006228796881594
    vertical_pixels: 540
    horizontal_pixels: 720